data/sql/updates/pending_db_world/rev_1687513981074967100.sql

```diff
@@ -0,0 +1,49 @@
+-- Hearts of the Pure Rp -------------
+
+-- The orientation of the initial NPC This is roughly towards the coordinates that need to be sniffed
+UPDATE `creature` SET `orientation`=4.41048 WHERE `guid`=41833;
+
+-- 3 emoticons when preparing for a task
+DELETE FROM `quest_details` WHERE `ID`=1476;
+-- EMOTE_ONESHOT_QUESTION && EMOTE_ONESHOT_LAUGH && EMOTE_ONESHOT_TALK
+INSERT INTO `quest_details` (`ID`, `Emote1`, `Emote2`, `Emote3`, `Emote4`, `EmoteDelay1`, `EmoteDelay2`, `EmoteDelay3`, `EmoteDelay4`, `VerifiedBuild`) VALUES
+(1476, 6, 11, 1, 0, 200, 0, 0, 0, 0);
+
+UPDATE `quest_offer_reward` SET `Emote1`=1 WHERE `ID`=1472;-- EMOTE_ONESHOT_TALK
+
+-- Use SmatAI
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 5693;
+
+-- Update emoticons
+UPDATE `creature_text` SET `Emote`=25 WHERE `CreatureID`=5693 AND `GroupID`=0 AND `ID`=0;-- EMOTE_ONESHOT_POINT
+UPDATE `creature_text` SET `Emote`=1 WHERE `CreatureID`=5693 AND `GroupID`=1 AND `ID`=0;-- EMOTE_ONESHOT_TALK
+UPDATE `creature_text` SET `Emote`=25 WHERE `CreatureID`=5693 AND `GroupID`=2 AND `ID`=0;-- EMOTE_ONESHOT_POINT
+UPDATE `creature_text` SET `Emote`=1 WHERE `CreatureID`=5693 AND `GroupID`=3 AND `ID`=0;-- EMOTE_ONESHOT_TALK
+UPDATE `creature_text` SET `Emote`=11 WHERE `CreatureID`=5693 AND `GroupID`=4 AND `ID`=0;-- EMOTE_ONESHOT_LAUGH
+
+-- Update unit_flags Make summoned NPCs unattackable and unselectable
+UPDATE `creature_template` SET `unit_flags` = 33555200 WHERE (`entry` = 5691);
+UPDATE `creature_template` SET `unit_flags` = 33555200 WHERE (`entry` = 5692);
```

Comment:
These are also sniffed and since we already need one we can take the flag values as well